File: Examen/C/ParOImpar.c
#include <stdio.h>

int main() {
    int numero;

    printf("Digite un numero entero: ");
    scanf("%d", &numero);

    if (numero % 2 == 0)
        printf("El numero %d es PAR.\n", numero);
    else
        printf("El numero %d es IMPAR.\n", numero);

    return 0;
}

File: Examen/Java/ParOImpar.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Other/File.java to edit this template
 */
import java.util.Scanner;
/**
 *
 * @author gyael
 */
public class ParOImpar {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese un numero entero: ");
        int n = sc.nextInt();

        if (n % 2 == 0)
            System.out.println("El numero es PAR.");
        else
            System.out.println("El numero es IMPAR.");

        sc.close();
    }
}

